builtin/providers/aws/provider.go

```diff
@@ -178,6 +178,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_elasticache_subnet_group":     resourceAwsElasticacheSubnetGroup(),
 			"aws_elb":                          resourceAwsElb(),
 			"aws_flow_log":                     resourceAwsFlowLog(),
+			"aws_glacier_vault":								resourceAwsGlacierVault(),
```

Comment:
Can you try `gofmt` all the files you changed to avoid things like this, please? :wink: